viewer/src/main/resources/config/defaultviewerconfig.properties

```diff
@@ -21,6 +21,7 @@ lineLength=5.0
 gapLength=15.0
 gapLengthExit=9.0
 backgroundColor=4AAC17
+backgroundPicturePath=/home/test.jpg
```

Comment:
This should not be the default as the resource is not available nor provided. 
Could you provide a simple example in the sim/ path to demonstrate the feature e.g. in combination with the spiral geometry?